### PR TITLE
feat: add pr overview to app

### PR DESCRIPTION
**Pull request by jochemloedeman**
Adding back the original pull requests page. It is configured as a custom redirect action when clicking the pull requests button on the home page.

### CODE REVIEW

File: .github/workflows/deploy_dashboard_to_github_pages.yml [hunk 1]
```diff
@@ -54,6 +54,9 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install duckdb pandas Jinja2
 
+      - name: Generate pull requests page
+        run: python pull_requests/generate_html.py ${{ env.DUCKDB_PATH }} public/pull_requests.html
```
Comment (JCZuurmond) [inline .github/workflows/deploy_dashboard_to_github_pages.yml]: Could you put the `generate_` scripts in the same location?

File: dashboard/pull_requests/pull_requests_template.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<!doctype html>
```
Comment (JCZuurmond) [inline dashboard/pull_requests/pull_requests_template.html]: Could you put the templates in the same documentation? And and with `.template`, like the other does.

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
@jochemloedeman : LGTM! Just a couple restructure comments

File: dashboard/generate/definitions.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from pathlib import Path
+
+
+TEMPLATES_PATH = Path(__file__).parents[1] / "templates"
+PULL_REQUESTS_TEMPLATE = "pull_requests.html.template"
+CONSTANTS_TEMPLATE = "constants.tsx.template"
```
Comment (jochemloedeman) [inline dashboard/generate/definitions.py:6]: Added a definitions file to help with the shared variables in the package

Comment (JCZuurmond) [reply dashboard/generate/definitions.py:6]: Allright! This LGTM, eventually we want to use `duckdb-wasm`

**Review by JCZuurmond — APPROVED** [review]

**Pull request merged**